interpretation/od_analyzer.py

```diff
@@ -0,0 +1,143 @@
+import pandas as pd
+import os
+import re
+from matplotlib import pyplot as plt
+import seaborn as sns
+from interpretation.plotting import roc_plot_grid, standard_curve_plot
+from interpretation.report_reader import slice_df, normalize_od, read_output_batch
+import array_analyzer.extract.constants as constants
+
+def read_config(input_dir):
+    """
+    Load analysis config from the input directory.
```

Comment:
My understanding is the spot fitting config reading is done by `MetaData` class, and `MetaData` calls `txt_parser` to convert the loaded dataframes into dictionaries. I thought about making this function a method in `MetaData` class but it doesn't seem to make much sense because I can't reuse most of the existing properties and methods in `MetaData`. Let me know if you have ideas on how to better merge the two meatadata/config readers.